src/test/java/com/arpnetworking/kairosdb/aggregators/HistogramFilterAggregatorTest.java

```diff
@@ -26,22 +27,45 @@
 import org.kairosdb.core.datastore.DataPointGroup;
 import org.kairosdb.testing.ListDataPointGroup;
 
-import java.util.Map;
 import java.util.TreeMap;
 
 /**
  * Unit tests for the Histogram Filter Aggregator.
  *
- * @author Joey Jackson
+ * @author Joey Jackson (jjackson at dropbox dot com)
  */
 public class HistogramFilterAggregatorTest {
     private HistogramFilterAggregator _aggregator;
 
-    private static final double THRESHOLD_POSITIVE_AT_BOUNDARY = Double.longBitsToDouble(0x4059000000000000L); //100.0
-    private static final double THRESHOLD_POSITIVE_NOT_AT_BOUNDARY = Double.longBitsToDouble(0x40590000FFFFFFFFL); //100.01
-    private static final double THRESHOLD_NEGATIVE_AT_BOUNDARY = Double.longBitsToDouble(0xc059000000000000L); //-100.0
-    private static final double THRESHOLD_NEGATIVE_NOT_AT_BOUNDARY = Double.longBitsToDouble(0xc0590000FFFFFFFFL); //-100.01
-
+    private static final double NEG_516_0 = Double.longBitsToDouble(0xc080200000000000L);  //-516.0
+    private static final double NEG_512_0 = Double.longBitsToDouble(0xc080000000000000L);  //-512.0
+    private static final double NEG_100_5 = Double.longBitsToDouble(0xc059200000000000L);  //-100.5
+    private static final double NEG_100_01 = Double.longBitsToDouble(0xc0590000FFFFFFFFL); //-100.01
+    private static final double NEG_100_0 = Double.longBitsToDouble(0xc059000000000000L);  //-100.0
+    private static final double NEG_99_5 = Double.longBitsToDouble(0xc058e00000000000L);   // -99.5
+    private static final double NEG_1EN308 = Double.longBitsToDouble(0x8000200000000000L); //  -1.e-308
+    private static final double NEG_0_0 = Double.longBitsToDouble(0x8000000000000000L);    //  -0.0
+    private static final double POS_0_0 = Double.longBitsToDouble(0x0000000000000000L);    //   0.0
+    private static final double POS_1EN308 = Double.longBitsToDouble(0x0000200000000000L); //   1.e-308
+    private static final double POS_99_5 = Double.longBitsToDouble(0x4058e00000000000L);   //  99.5
+    private static final double POS_100_0 = Double.longBitsToDouble(0x4059000000000000L);  // 100.0
+    private static final double POS_100_01 = Double.longBitsToDouble(0x40590000FFFFFFFFL); // 100.01
+    private static final double POS_100_5 = Double.longBitsToDouble(0x4059200000000000L);  // 100.6
+    private static final double POS_512_0 = Double.longBitsToDouble(0x4080000000000000L);  // 512.0
+    private static final double POS_516_0 = Double.longBitsToDouble(0x4080200000000000L);  // 516.0
```

Comment:
A couple of these constants aren't the same as their commented value: https://www.dropbox.com/s/060cb3wql8o7pbz/Screenshot%202019-06-14%2010.30.51.png?dl=0
An alternative might be to use `truncate` for the bin-aligned values, and... adding small deltas, or maybe `(truncate+binBound)/2`, for the non-bin-aligned values? Like
```
    private static final double NEG_100_0 = truncate(-100.0);
    private static final double NEG_99_5 = (truncate(-99.5) + binBound(-99.5)) / 2;
```
or
```
    private static final double NEG_100_0 = truncate(-100.0);
    private static final double NEG_99_5 = NEG_100_0 - 0.5;
```
or something else entirely, as long as the constants' names are more "obviously correct"?